components/installer/pkg/kymainstallation/stepsfactory.go

```diff
@@ -51,8 +53,13 @@ func NewStepFactory(kymaPackage kymasources.KymaPackage, helmClient kymahelm.Cli
 	}
 
 	if relesesRes != nil {
+		log.Println("Helm releases list:")
```

Comment:
consider making this line more visible in logs. Atm it is very hard to spot this list in the logs.